app/pathogen/sarscov2/analyze/columns.tsx

```diff
@@ -1,137 +1,137 @@
-"use client";
```

Comment:
Shoutout to the literal worst git diff I have ever seen lmao

Reply:
wtf ahahahahah why did this happen